torchsparse/nn/modules/conv.py

```diff
@@ -51,14 +59,16 @@ def __init__(self,
             assert not transpose
 
     def __repr__(self):
```

Comment:
Same here. We only need to define `extra_repr` for a new `nn.Module` (see https://pytorch.org/docs/stable/notes/extending.html#adding-a-module for more details).